examples/multistep_workflow/conda.yaml

```diff
@@ -4,8 +4,9 @@ channels:
 dependencies:
   - python=3.6
   - pyspark
-  - keras
   - requests
   - click
   - pip:
-    - mlflow
+    - tensorflow==1.13.1
+    - keras==2.2.4
```

Comment:
why did this need to be moved under pip do you know?

Reply:
Good question I should have put a comment.  

The mkl version of tensorflow from conda and Spark don't like each other. There is a library that ends up loaded twice, complains and kills the program. You can get around by installing nomkl package from conda (seems shady) or setting an environment variable instructing the library to allow this behavior (not recommended by the authors). 

I think the safest and cleanest is to install from pip.